lms/djangoapps/grades/tests/test_new.py

```diff
@@ -168,18 +168,18 @@ def test_create_internals(self):
             with patch(
                 'lms.djangoapps.grades.new.subsection_grade.SubsectionGradeFactory._get_bulk_cached_grade',
                 wraps=self.subsection_grade_factory._get_bulk_cached_grade
-            ) as mock_get_saved_grade:
+            ) as mock__get_bulk_cached_grade:
```

Comment:
That was intentional, because the mocked method is called `_get_bulk_cached_grade`.